src/fs/driver/devfs/devfs_oldfs.c

```diff
@@ -59,3 +60,40 @@ static const struct fs_driver devfs_driver = {
 };
 
 DECLARE_FILE_SYSTEM_DRIVER(devfs_driver);
+
+int devfs_add_block(struct block_dev *bdev) {
+	struct path node, root;
+	struct nas *nas;
+	struct node_fi *node_fi;
+	char full_path[PATH_MAX];
+
+	vfs_get_root_path(&root);
+
+	/* XXX  Workaround for compatibility with DVFS.
```

Comment:
I think that this comment is irrelevant now and can be deleted.

Reply:
done